src/test/java/Homework23/LoginPage23.java

```diff
@@ -0,0 +1,49 @@
+package Homework23;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginPage23 extends BasePage23 {
+
+    @FindBy(css="[type='submit']")
+            WebElement submitButtonLocator;
+
+    @FindBy(css = "[type='email']")
+            WebElement emailField;
+
+    @FindBy(css = "[type='password']")
+    WebElement passwordField;
+
+    //locators
+  //  By submitButtonLocator = By.cssSelector("[type='submit']");
+    //By emailField = By.cssSelector("[type='email']");
+    //By passwordField = By.cssSelector("[type='password']");
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.